### DISCUSSION TITLE
总部答应退款，广州饶铭只退1元：安卓手机没法用，电脑网页登录 chatgpt 老是要验证是否是机器人

### DISCUSSION THREAD

**Discussion by luoposhanhuige** [getlantern/lantern-forum-cn#1081] [General]
<img width="488" alt="截屏2024-12-12 12 16 35" src="https://github.com/user-attachments/assets/81862e02-a1df-4def-986e-956ecacf8628" />

之前提交过，但没人搭理。特此在论坛中申请，希望管理员协助。谢谢！
补充说明：账号在附件截图中。

**Comment by MaxMKMax** [comment]
@luoposhanhuige 您好. 好的, 我会帮您处理. 不好意思啊, 客服那里不是我负责的, 所以看不到您的信息.
还有您可以提供我单据吗? 这样我才可以帮您提交问题上去. 谢谢您!

**Reply by luoposhanhuige** [reply]
谢谢管理员，我的单据如下附件：
![IMG1202](https://github.com/user-attachments/assets/e254904d-1ae9-4225-bcf1-b84c6b29448e)

**Comment by MaxMKMax** [comment]
@luoposhanhuige 在帮您提交退款之前需要您先帮个忙我们才能进行退款. 我们也想找出原因为什么chatgpt 老是要验证是否是机器人.

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Comment by MaxMKMax** [comment]
@luoposhanhuige 您好. 我刚才和技术团队开会了. 也深入研究您遇到的问题了. 
技术团队说是因为美国的OpenAi (ChatGPT) 的技术Proxy 问题. 他们没有开放流通. 所以无论用哪一款的VPN也会有同样的问题. 需要您向OpenAi 他们反馈一下中国用户遇到的情况. 只要他们那里解决了, 大家就不会遇到这个问题了. 希望您理解. 谢谢您啊!

**Comment by MaxMKMax** [comment]
@luoposhanhuige 在帮您提交退款之前需要您先帮个忙我们才能进行退款. 我们也想找出原因为什么chatgpt 老是要验证是否是机器人.

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Comment by luoposhanhuige** [comment]
退款流程进展如下，国际总部答复退款。但通过支付宝联系广州饶铭公司，暂时没有下文，不知道该联系谁？请求再次协调帮助。
附上两份截图：
![1](https://github.com/user-attachments/assets/77b57a95-13b9-4cc1-9cb2-ae9a6ce239dd)
![2](https://github.com/user-attachments/assets/7e81feb2-f005-478f-865e-b258ad941d78)

**Reply by luoposhanhuige** [reply]
希望得到你的再次及时协调，谢谢！

**Comment by luoposhanhuige** [comment]
昨天收到广州饶铭退到支付宝的1元钱，太搞笑了，哈哈。网络上找不到饶铭公开的电话，没法进一步联系，这事就这么不了了之了？

![3](https://github.com/user-attachments/assets/8c9a4587-0f20-4bec-8d92-b55f81311dd9)
![4](https://github.com/user-attachments/assets/f7ca5b0c-2646-4045-a934-4e7b1b1b49c0)
